src/main/java/org/folio/helper/CheckinHelper.java

```diff
@@ -74,13 +72,12 @@ public Future<ReceivingResults> checkinPieces(CheckinCollection checkinCollectio
   }
 
   private Future<ReceivingResults> processCheckInPieces(CheckinCollection checkinCollection, RequestContext requestContext) {
-    Map<String, Map<String, Location>> pieceLocationsGroupedByPoLine = groupLocationsByPoLineIdOnCheckin(checkinCollection);
-     // 1. Get piece records from storage
+    // 1. Get piece records from storage
     return createItemsWithPieceUpdate(checkinCollection, requestContext)
       // 2. Filter locationId
       .compose(piecesByPoLineIds -> filterMissingLocations(piecesByPoLineIds, requestContext))
```

Comment:
if we delete invoking groupLocationsByPoLineIdOnCheckin() than piecesByPoLineIds should be renamed, also piecesGroupedByPoLine in updateInventoryItemsAndHoldings()

Reply:
Why we need to rename piecesByPoLineIds?